ledger/complete/mtrie/flattener/iterator.go

```diff
@@ -66,7 +67,7 @@ type NodeIterator struct {
 // as for each node, the children have been previously encountered.
 func NewNodeIterator(mTrie *trie.MTrie) *NodeIterator {
 	// for a Trie with k := mTrie.KeyLength() [bytes], the longest possible path can contain at most 8k+1 vertices
-	stackSize := mTrie.PathLength()*8 + 1
+	stackSize := hash.TreeMaxHeight + 1
```

Comment:
I think mTrie.PathLength()*8 + 1 == TreeMaxHeight, so we might rename TreeMaxHeight to something else. max traverse len?